server/api/after_auth_handler.py

```diff
@@ -20,9 +21,18 @@ def authorize():
         g_user = gauth.google.userinfo()
         db_user = user.User(g_user).save("users")
```

Comment:
Is this right? I'm not sure which functions this is calling

Reply:
I think it's creating a new user using the data from the g_user variable and then calling save?
You should be able to do:
`db_user = User(g_user).save("users")`

Reply:
no. user is name of the module where User class exist.